components/tidb_query_vec_expr/src/impl_cast.rs

```diff
@@ -2110,8 +2110,8 @@ mod tests {
     fn test_cast_real_time() {
         let cases = vec![
             ("2019-09-16 10:11:12", 190916101112.111, 0),
-            ("2019-09-16 10:11:12", 20190916101112.111, 0),
-            ("2019-09-16 10:11:12", 20190916101112.123, 0),
+            ("2019-09-16 10:11:12", 20_190_916_101_112.11, 0),
```

Comment:
Suppress this kind of lint also.

Reply:
Because 1) the big numbers are usually in tests 2) the underscores make the number unobvious.

Reply:
Fixed and removed all "_" in numbers introduced by previous changes